src/components/FeaturePanel/Climbing/ClimbingRestriction.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Alert, AlertTitle } from '@mui/material';
+import { useFeatureContext } from '../../utils/FeatureContext';
+import { t } from '../../../services/intl';
+
+export const ClimbingRestriction = () => {
+  const { feature } = useFeatureContext();
```

Comment:
I would like this condition here in the very top. It must clear for first time what is the main condition.

```suggestion
  const { feature } = useFeatureContext();
  if (!feature.tags.climbing) {
    return null;
  }
```